pandas/tests/test_groupby.py

```diff
@@ -4594,9 +4594,6 @@ def test_frame_groupby_plot_boxplot(self):
         _skip_if_mpl_not_installed()
 
         import matplotlib.pyplot as plt
-        import matplotlib as mpl
```

Comment:
I mean you can put the line in the existing `_skip_if_mpl_not_installed`. So adapt `_skip_if_mpl_not_installed` to import matplotlib and call `mpl.use('Agg')`, and then import pyplot here in the testµµ

I also don't exactly know why it is needed (maybe to run the tests where no graphical toolkit is avaliable?), but since all existing tests for plotting do this, I would keep it that way.